src/platform/utilities/sso/keepAliveSSO.js

```diff
@@ -53,9 +58,14 @@ export default async function keepAlive() {
       cache: 'no-store',
     });
 
-    await resp.text();
     const alive = resp.headers.get(AUTHN_HEADERS.ALIVE);
 
+    // If no CSP or session-alive headers, return early
+    if (resp.headers.get(AUTHN_HEADERS.CSP) === undefined || alive !== 'true') {
```

Comment:
Just want to verify `resp.headers.get(AUTHN_HEADERS.CSP)` cannot ever be `null`? If it can, `null === undefined` will be `false`

Reply:
Good catch.  I think I just wanted `null` not undefined